mapie/tests/test_classification.py

```diff
@@ -681,7 +682,10 @@ def test_valid_method(method: str) -> None:
     check_is_fitted(mapie_clf, mapie_clf.fit_attributes)
 
 
-@pytest.mark.parametrize("cv", [None, -1, 2, KFold(), LeaveOneOut()])
+@pytest.mark.parametrize(
+    "cv", [None, -1, 2, KFold(), LeaveOneOut(),
+           ShuffleSplit(n_splits=1, test_size=0.1), "prefit"]
+)
 def test_valid_cv(cv: Any) -> None:
     """Test that valid cv raises no errors."""
     model = LogisticRegression(multi_class="multinomial")
```

Comment:
Why no test is implemented to test the result ? (like you did for regression)

Reply:
ok, let's add them

Reply:
It is done in classification with this test `test_toy_dataset_predictions` which loops over strategies

Reply:
It was more difficult than expected: it was necessary to review the management of the random_state with the cv method (in connection with the Issue #290). To answer your request, I solved the problem directly but it will be necessary to complete the code with additional tests (future PR of Issue #290). Therefore, I added more changes than expected.

Reply:
I have detected a side-effect with `cv=split`. I have made a correction. Now, the `test_toy_dataset_predictions` is also done for `cv=split`.